client/state/utils/actions.js

```diff
@@ -0,0 +1,12 @@
+/**
+ * Internal dependencies
+ */
+import { UTILS_UNIQUEID_INCREMENT } from 'state/action-types';
+
+export function uniqueId( prefix ) {
```

Comment:
Yes, it is exactly to mimic `lodash`, it only gets returned prepended to the new id :)